index.html

```diff
@@ -687,35 +689,30 @@ <h1>Streaming GCN<br>Notices in Python</h1>
 ---
 
 ## We’re planning lots of enhancements in the near future:
-* Modernization of GCN Circulars
+* New alert types and alerts from new missions and facilities
 * Unified schema and alert format for GCN Kafka
+* Modernization of GCN Circulars
 * Integrated, searchable database of Notices and Circulars (GCN Viewer)
 * Interoperability with other transient Kafka brokers (e.g. [SCiMMA](https://scimma.org))
-* Alerts from new missions and facilities
 </textarea>
 </section>
 
-    <section>
-      <div class="image-and-caption">
-        <div>
-          <img src="img/screen-shot-gcn-markdown.png" width="700" alt="Screen shot of GCN Circular markdown">
-        </div>
-        <div data-markdown>
-          <textarea data-template>
-            ## Modernization of GCN Circulars
-            * Manage your own subscriptions
-            * Self-register to submit using peer endorsement
-            * Submit circulars by web form, API, or email
-              * Correct handling of Unicode
-              * "Astro Flavored Markdown"
-                * Annotate and cross-link references, dates, times, coordinates
-                * Embedded data tables, image attachments, math mode
-            * Automatically mint DOIs and BiBTeX entries
-            * Link your ORCID to your GCN Circulars
-          </textarea>
-        </div>
-      </div>
-    </section>
+<section>
+  <div class="image-and-caption">
+    <div>
+      <img src="img/screen-shot-new-producers.png" width="800" alt="Screen shot of New Notice Producers Documentation">
+    </div>
+    <div data-markdown>
+      <textarea data-template>
+        ## Create New Notice Types
```

Comment:
For the most part, we aren't using title case for slide headings.

```suggestion
        ## Create new Notice types
```